src/inmanta/server/services/compilerservice.py

```diff
@@ -32,7 +32,7 @@
 from itertools import chain
 from logging import Logger
 from tempfile import NamedTemporaryFile
-from typing import Optional, cast
+from typing import Optional, Type, cast
```

Comment:
Actual change request: pretty sure this is deprecated in favor of built-in `type`.